docs/api.md

```diff
@@ -226,7 +226,9 @@ Closes browser with all the pages (if any were opened). The browser object itsel
 - returns: <[string]> Browser websocket url.
 
 Browser websocket endpoint which could be used as an argument to
-[puppeteer.connect](#puppeteerconnectoptions).
+[puppeteer.connect](#puppeteerconnectoptions). The format is `ws://${host}:${port}/page/<id>`
```

Comment:
ironically, this is page's debugging URL, not browser's. It might work for now, but might not eventually. 

The Browser WS is dumped by the browser in its stderr.Note: this only happens if the browser is launched with a `remote-debugging-port` flag.

Check out how we extract browser web socket from stderr in the [Launcher.js](https://github.com/GoogleChrome/puppeteer/blob/d7e673645af749bcdf104abe79829c9720c420a0/lib/Launcher.js#L137)